### ISSUE TITLE
Tests with Duration

### ISSUE THREAD

**Issue by mohamede1945** [zalando/SwiftMonkey#29]
Since running tests forever is not really practical for some cases. Is it possible to add a method to test for a specified duration. Something like that

```swift
extension Monkey {
    func startTesting(forDuration duration: TimeInterval) {
        let start = CFAbsoluteTimeGetCurrent() // start time

        repeat {
            actRandomly()
            actRegularly()
        } while ((CFAbsoluteTimeGetCurrent() - start) < duration)
    }
}
```

Then we can call it that way:
```swift
// Run the monkey tests for 15 minutes
monkey.startTesting(forDuration: 15 * 60)

// or to fall back to forever testing
monkey.startTesting(forDuration: .infinity)
```

**Comment by cakushin** [comment]
What is the monkey click frequency?

**Comment by wojciechczerski** [comment]
Solved by #43 - closing the issue.
